L1Trigger/RPCTriggerPrimitives/BuildFile.xml

```diff
@@ -0,0 +1,37 @@
+<export>
+  <lib name="1"/>
+</export>
+
+<use name="root"/>
```

Comment:
@maseguracern @rekovic is this dependency on root really needed here? Where is it used? @smuzaffar FYI